### PR TITLE
[expo-image-picker] Bump cropper version to fix eas build

### PR DESCRIPTION
**Pull request by alexandrius**
# Why
Android Image Cropper lib manifest includes multiple `<activity/>` elements in the manifest. They actually removed those in the [next version](https://github.com/CanHub/Android-Image-Cropper/compare/4.3.0...4.3.1). The activity elements included style tag - causing app build fail on SDK49-Beta:

```
[stderr] 
/home/expo/workingdir/build/android/app/src/debug/AndroidManifest.xml:41:13-49 Error:
[stderr] 
	Attribute activity#androidx.test.core.app.InstrumentationActivityInvoker$BootstrapActivity@theme value=(@android:style/Theme) from [com.github.CanHub:Android-Image-Cropper:4.3.0] AndroidManifest.xml:41:13-49
[stderr] 
	is also present at [androidx.test:core:1.5.0] AndroidManifest.xml:30:13-56 value=(@style/WhiteBackgroundTheme).
[stderr] 
	Suggestion: add 'tools:replace="android:theme"' to <activity> element at AndroidManifest.xml:38:9-45:20 to override.
[stderr] 
/home/expo/workingdir/build/android/app/src/debug/AndroidManifest.xml:49:13-49 Error:
[stderr] 
	Attribute activity#androidx.test.core.app.InstrumentationActivityInvoker$EmptyActivity@theme value=(@android:style/Theme) from [com.github.CanHub:Android-Image-Cropper:4.3.0] AndroidManifest.xml:49:13-49
[stderr] 
	is also present at [androidx.test:core:1.5.0] AndroidManifest.xml:38:13-56 value=(@style/WhiteBackgroundTheme).
[stderr] 
	Suggestion: add 'tools:replace="android:theme"' to <activity> element at AndroidManifest.xml:46:9-53:20 to override.
[stderr] 
/home/expo/workingdir/build/android/app/src/debug/AndroidManifest.xml:57:13-56 Error:
[stderr] 
	Attribute activity#androidx.test.core.app.InstrumentationActivityInvoker$EmptyFloatingActivity@theme value=(@android:style/Theme.Dialog) from [com.github.CanHub:Android-Image-Cropper:4.3.0] AndroidManifest.xml:57:13-56
[stderr] 
	is also present at [androidx.test:core:1.5.0] AndroidManifest.xml:46:13-62 value=(@style/WhiteBackgroundDialogTheme).
[stderr] 
	Suggestion: add 'tools:replace="android:theme"' to <activity> element at AndroidManifest.xml:54:9-61:20 to override.
```

# How
Changed 4.3.0 to 4.3.1

# Test Plan

- Install the library in an empty project
- Build the app

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-image-picker/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "expo-image-picker",
-  "version": "14.3.0",
+  "version": "14.3.1",
```
Comment (alanjhughes) [inline packages/expo-image-picker/package.json:3]: Revert this please. This will be done by the publish scripts.

**Review by alanjhughes — CHANGES_REQUESTED** [review]
Thank you for your contribution 👍 Just one change.

File: packages/expo-image-picker/package.json (hunk 1 above)
Comment (alexandrius) [reply packages/expo-image-picker/package.json]: done

File: docs/react-native-website
Comment (alanjhughes) [inline docs/react-native-website:1]: This should not be here either

**Review by alanjhughes — CHANGES_REQUESTED** [review]

File: packages/expo-image-picker/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,9 @@
 
 ### 💡 Others
 
+## 14.3.1 — 2023-06-29
+- Fix manifest merger build fail on Android. ([#23191](https://github.com/expo/expo/pull/23191) by [@alexandrius](https://github.com/alexandrius))
```
Comment (tsapeta) [inline packages/expo-image-picker/CHANGELOG.md:14]: Can you please remove the version and date from here and move the entry a few lines up to be under bug fixes section?

**Review by tsapeta — CHANGES_REQUESTED** [review]

File: packages/expo-image-picker/CHANGELOG.md (hunk 2 above)
Comment (alexandrius) [reply packages/expo-image-picker/CHANGELOG.md]: Sorry for these. Will be more careful next time.

**Review by alanjhughes — APPROVED** [review]
Thanks!

**Review by tsapeta — APPROVED** [review]

**Comment by superyarik** [comment]
hey @alexandrius can this issue fixed in SDK49 [affect](https://github.com/expo/expo/issues/24467) my EAS Build with SDK48? thanks

**Pull request merged**